backend/lib/dbutils.js

```diff
@@ -15,8 +15,8 @@ async function checkMongoWorks(beFatal) {
     } catch(error) {
         debug("Failure in checkMongoWorks: %s", error.message);
         if(beFatal) {
-            console.log("mongodb is not running: quitting");
```

Comment:
this matter because someone might run `node bin/server` and without `console.log` it would quit silently, I'll revert later with the allow console

Reply:
restored